WalletWasabi.Fluent/Styles/TextBox.axaml

```diff
@@ -153,15 +154,17 @@
                            CaretBrush="{TemplateBinding CaretBrush}"
                            HorizontalAlignment="{TemplateBinding HorizontalContentAlignment}"
                            VerticalAlignment="{TemplateBinding VerticalContentAlignment}" />
-          </Panel>
         </ScrollViewer>
       </ControlTemplate>
     </Setter>
   </Style>
 
-  <Style Selector="TextBlock.monoSpaced, TextBox.monoSpaced">
+  <Style Selector="TextBlock.monoSpaced, c|PrivacyContentControl.monoSpaced">
+    <Setter Property="(TextBlock.FontFamily)" Value="{StaticResource MonospacedFont}" />
+  </Style>
```

Comment:
In `PrivacyContentControl.axaml` there is already a `monoSpaced` class. Could you take a look? It might be simplified.
Also the monospaced style class for the PrivacyContentControl should not be in the TextBox.axaml.